.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
what is this for?

Reply:
No idea it was generated by storybook's `npx sb init`

Reply:
ok 🙂

Reply:
I'll try to remove it and see if anything breaks 🤷‍♀️

Reply:
No this is fine if this is coming from the official installation method 🙂

Reply:
https://storybook.js.org/docs/react/configure/overview#configure-story-rendering